articles/operator-nexus/howto-configure-cluster.md

```diff
@@ -155,6 +155,70 @@ az networkcloud cluster deploy \
   --subscription "$SUBSCRIPTION_ID"
 ```
 
+> [!TIP]
+> To check the status of the `az networkcloud cluster deploy` command, it can be executed using the `--debug` flag.
+> This will allow you to obtain the `Azure-AsyncOperation` or `Location` header used to query the `operationStatuses` resource.
+> See the section [Cluster Deploy Failed](#cluster-deploy-failed) for more detailed steps.
+> Optionally, the command can run asynchronously using the `--no-wait` flag.
+
+### Cluster Deploy and Hardware Validation
+
+During a Cluster deploy process, one of the steps executed is hardware validation.
```

Comment:
should the word "optional" be added in this sentence? I don't believe its' required and showing how it's on or off, would be good.